File: prime.py
if number > 1:
    for itr in range(2, int(number/2)+1):
        if (number % itr) == 0:
            print(number, "is NOT a PRIME number")
            break
        else:
            print(number, "is a PRIME number") 
else:
    print(number, "is NOT a PRIME number")

File: Java_Programs_for_beginners/JavaNumber_Programs/NumberSystem/PrimeNumber/prime.java
import java.util.*;
public class prime {
    public static void main (String args[]){
        Scanner in = new Scanner(System.in);
        int a,n,c=0;
        System.out.println("Enter a number");
        n=in.nextInt();
        for(a=1;a<=n;a++){
            if(n%a==0)
            c++;
        }
        if(c==2)
        System.out.println("prime number");
        else
        System.out.println("Not a prime number");
    }
}
